### PR TITLE
Release v2.6.2

### PR DESCRIPTION
**Pull request by maheshwaghmare**
- Improvement: Used better and optimised way to show the initial sites and pages list on Starter Template dashboard.
- Improvement: Block Templates - Used version.json instead of package.json to manage the dependency.
- Improvement: Block Templates - Now users can suggest their required templates via suggestion link.
- Improvement: Block Templates - Added filter to disable the Block Templates. - https://wpastra.com/docs/filter-to-remove-starter-templates-button/
- Fix: Block Templates - Retained the selected block pattern category filter after switching between Pages/Patterns tabs.
- Fix: Rectified the invisible icon from the 'View Site' button.
- Fix: Brizy pages initially not showing any content.

### CODE REVIEW

File: inc/lib/ast-block-templates/classes/class-ast-block-templates.php [hunk 1]
```diff
@@ -377,18 +377,22 @@ public function template_assets() {
...
+						'wpforms_status'          => $this->get_plugin_status( 'wpforms-lite/wpforms.php' ),
+						'gutenberg_status'        => $this->get_plugin_status( 'gutenberg/gutenberg.php' ),
+						'_ajax_nonce'             => wp_create_nonce( 'ast-block-templates-ajax-nonce' ),
+						'button_text'             => esc_html__( 'Starter Templates', 'ast-block-templates' ),
+						'display_button_logo'     => true,
+						'popup_logo_uri'          => AST_BLOCK_TEMPLATES_URI . 'dist/logo.svg',
+						'button_logo'             => AST_BLOCK_TEMPLATES_URI . 'dist/starter-template-logo.svg',
+						'button_class'            => '',
+						'display_suggestion_link' => true,
+						'suggestion_link'         => 'https://wpastra.com/sites-suggestions/?utm_source=demo-import-panel&utm_campaign=astra-sites&utm_medium=suggestions',
```
Comment (vrundakansara) [inline inc/lib/ast-block-templates/classes/class-ast-block-templates.php:395]: This link is defined in 2 places. Can we have a function/constant for it?

File: inc/lib/ast-block-templates/classes/class-ast-block-templates.php [hunk 2]
```diff
@@ -377,18 +377,22 @@ public function template_assets() {
...
+						'ajax_url'                => admin_url( 'admin-ajax.php' ),
+						'uri'                     => AST_BLOCK_TEMPLATES_URI,
+						'white_label_name'        => '',
+						'allBlocks'               => $this->get_all_blocks(),
+						'allSites'                => $this->get_all_sites(),
+						'wpforms_status'          => $this->get_plugin_status( 'wpforms-lite/wpforms.php' ),
+						'gutenberg_status'        => $this->get_plugin_status( 'gutenberg/gutenberg.php' ),
+						'_ajax_nonce'             => wp_create_nonce( 'ast-block-templates-ajax-nonce' ),
+						'button_text'             => esc_html__( 'Starter Templates', 'ast-block-templates' ),
+						'display_button_logo'     => true,
```
Comment (vrundakansara) [inline inc/lib/ast-block-templates/classes/class-ast-block-templates.php:390]: Do you think we should add a filter for this flag?

**Review by vrundakansara — COMMENTED** [review]
@amithire1201 - Please add these suggestions as a task in the next update.

File: inc/lib/ast-block-templates/classes/class-ast-block-templates.php (hunk 1 above)
Comment (amithire1201) [reply inc/lib/ast-block-templates/classes/class-ast-block-templates.php:395]: Task added - https://trello.com/c/ZHuX3ZAm/208-suggestion-links-improvement

File: inc/lib/ast-block-templates/classes/class-ast-block-templates.php (hunk 2 above)
Comment (amithire1201) [reply inc/lib/ast-block-templates/classes/class-ast-block-templates.php:390]: Task added - https://trello.com/c/H56Zp1JF/209-filter-for-display-logo-flag

File: inc/includes/admin-page.php [hunk 3]
```diff
@@ -730,25 +750,39 @@
...
 				<div class="install-theme-info">
 					<div class="astra-sites-advanced-options-wrap">
+
+						<?php if ( true === $subscription_status ) : ?>
+							<p><?php esc_html_e( 'To serve more beautiful starter templates, we would like to know more about you:', 'astra-sites' ); ?></p>
+						<?php endif; ?>
+
+						<div id="astra-sites-subscription-form-one" class="subscription-form astra-sites-subscription-form-one"></div>
+
+						<h2 class="astra-sites-advanced-options-heading">Advanced Options <span class="dashicons dashicons-arrow-down-alt2"></span></h2>
```
[the remark this answers is not in the document]
Comment (vrundakansara) [reply inc/includes/admin-page.php:760]: @amithire1201 - Please add a task for this too. I am marking it as resolved for now.

[the remark this answers is not in the document]
Comment (amithire1201) [reply inc/includes/admin-page.php:760]: Task added - https://trello.com/c/Gi5Qgbn3/210-translation-ready-strings

**Pull request merged**